resources/src/main.rs

```diff
@@ -65,10 +65,18 @@ async fn intercept_command(body: &Value) {
     let channel: &str = body["event"]["channel"]
         .as_str()
         .unwrap_or("invalid_channel");
+    if channel != "C0351GJ62Q0" {
+        log::info!("This channel is not the test channel");
+        return;
+    }
     log::info!("text: {}, channel: {}", text, channel);
 
     let lowercase_text: &str = &*text.to_lowercase();
 
+    if lowercase_text.contains("buns") {
```

Comment:
Do we want it to be any instance containing buns?
If so is there a more fun way to do it like the match statement below? 👀

Reply:
at first I was looking at doing any instance of buns so I could hopefully include :buns . That did not work, but it does add a feature of using buns in a sentence.